modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/BPlusTree.java

```diff
@@ -1529,6 +1563,32 @@ public final <R> R findOne(L row, TreeRowClosure<L, T> c, Object x) throws Ignit
         return findOne(row, null, null);
     }
 
+    /**
+     * Searches for the random row.
+     *
+     * @return random row.
+     * @throws IgniteCheckedException If failed.
+     */
+    public final @Nullable <R> R findRandom(TreeRowMapper<L, T, R> mapper) throws IgniteCheckedException {
```

Comment:
It would be great to write simple unit tests for this method, at your discretion.

Reply:
I think it's enough to have current tests. This method is quite trivial, and I also just copied everything from GG9 :)